### ISSUE TITLE
Add (optional) configuration for HTTPS

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/docker#14]
Document or configure nginx to support HTTPS, ideally we would support retrieving/renewing certificates from LetsEcrypt out of the box

**Comment by lucanello** [comment]
You could solve it using a different approach: like with other big software (e.g. Grafana, Nextcloud etc.), make wger using a reverse proxy like Traefik. I tried installing wger without the nginx-proxy and it worked but was massive headache. If you simply serve wger using the app-container without any more configuration, it would be no hassle to setup a SSL-secured application.

**Comment by rolandgeider** [comment]
yeah, that would be a possibility as well. The problem is that we need some kind of web server for the static files. Or do you mean putting traefik in front on nginx?
